bpf/lib/common.h

```diff
@@ -453,6 +453,29 @@ enum {
 #define METRIC_SERVICE  3
 
 /* Magic ctx->mark identifies packets origination and encryption status.
+ *
+ *  1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2
+ * +-------------------------------+---+---+-------+---------------+
+ * |L L L L L L L L L L L L L L L L|R R|E E|M M M M|U U U U U U U U|
+ * +-------------------------------+---+---+-------+---------------+
+ *  identity                        k8s     mark    identity
+ *
+ * Identity (24 bits):
+ * +-----------------------------------------------+
+ * |U U U U U U U U|L L L L L L L L L L L L L L L L|
+ * +-----------------------------------------------+
+ *  1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4
+ *
+ * Kubernetes Mark (2 bits):
+ * 0 1  Masquerade
+ * 1 0  Drop
+ *
+ * Cilium Mark (4 bits):
+ * M M M M
+ * (see MARK_MAGIC_* in bpf/lib/common.h)
+ *
+ * Encryption ID (2 bits):
+ * E E  Key ID
```

Comment:
As I noted out-of-band while Github was down, the 0x2000 bit will collide with portmap CNI so we still have that overlap. For that I think we're fine to just state an incompatibility between combining ipsec with portmap CNI (suggesting users to instead use kube-proxy-replacement). [Reference](https://github.com/fwmark/registry/#bitwise-mark-registry)

Reply:
Added